frontend/packages/dev-console/src/components/import/image-search/ImageStreamTagDropdown.tsx

```diff
@@ -70,7 +70,8 @@ const ImageStreamTagDropdown: React.FC<{
           setFieldValue(`${fieldPrefix}isi.tag`, selectedTag);
           setFieldValue(`${fieldPrefix}isi.ports`, ports);
           setFieldValue(`${fieldPrefix}image.ports`, ports);
-          formType !== 'edit' &&
+          formType &&
```

Comment:
@Lucifergene `formType` might not be set in every form that uses this component. Please check and make sure name is generated for these forms and the behaviour is not changed.

Reply:
@Lucifergene did you check this?

Reply:
Yes I removed this modification